Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -25,12 +25,15 @@
     <DefineConstants>$(DefineConstants);</DefineConstants>
     <UseFullSemVerForNuGet>false</UseFullSemVerForNuGet>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-    <PackageLicenseUrl>https://go.microsoft.com/fwlink/?linkid=868961</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
     <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
     <PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=868960</PackageProjectUrl>
     <MDocDocumentationDirectory>$(MSBuildThisFileDirectory)..\docs\en</MDocDocumentationDirectory>
     <Configurations>Debug;Release;Samples;Docs</Configurations>
     <DebugType>portable</DebugType>
+    <IncludeSymbols>true</IncludeSymbols> 
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat> 
```

Comment:
why did you include symbols? this might be nice, but was not requested; I would prefer we include this in a different PR

Reply:
This is a funny one. It is an attempt to circumvent a bug. When the script attempts to upload the package, it does: `dotnet nuget push Output\*.nupkg`. It turns out that when you specify packages with a `*`wildcard and finds a package, it automatically searches for a .snupkg - and exits with failure if it doesn't find it. This is a known bug that for some reason hasn't been fixed, and it does not happen when you specify the name of the package directly.

The only solution possible (other than generate the .snupkg) is to specify the package name directly, by searching the file and extracting it in scripts/nuget.fsx. I chose the simplest solution, (because I don't know F# and because I expect the nuget bug will get solved some day), but I can do the more complex one if necessary.

Reply:
well, it was you who introduced the wildcard :) I mean, the original nuget.fsx script you copied this from didn't have a wildcard, precisely for this reason! I'd prefer you remove the wildcard and remove the snupkg generation too.

Reply:
This one is going to take me more time than expected, as the old script code can't be reused and I have to learn the basics of F# to attempt to pass the string filename to nupkg; unfortunately F# is not very kind to newcomers...

Reply:
try this:
```
        let outputDir = Path.Combine(Directory.GetCurrentDirectory(), "Output") |> DirectoryInfo
        let nupkgFile = outputDir.EnumerateFiles().Single(fun file -> file.FullName.EndsWith ".nupkg")
```

and later:

```
        let Arguments = sprintf "nuget push %s -k %s -s %s"
                            nupkgFile.FullName nugetApiKey defaultNugetFeedUrl
```

Reply:
(not forgetting to put this at the top: 
```
open System.IO
open System.Linq
```
)

Reply:
Done, many thanks for your help.